src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -637,7 +637,8 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
             this TypeSymbol type,
             Func<TypeSymbol, T, bool, bool> predicate,
             T arg,
-            bool canDigThroughNullable = false)
+            bool canDigThroughNullable = false,
+            bool visitCustomModifiers = false)
```

Comment:
>bool visitCustomModifiers = false [](http://example.com/codeflow?start=12&length=33)

The parameter isn't used. Remove?

Reply:
Oh, it's a mistake, I forgot to pass it to the other overload.

Reply:
> Oh, it's a mistake, I forgot to pass it to the other overload.

Since none of the tests failed, I guess this parameter isn't really used and can be removed.

Reply:
We have missing tests.